src/data/seq/seq.lean

```diff
@@ -65,7 +69,7 @@ def head (s : seq α) : option α := nth s 0
 
 /-- Get the tail of a sequence (or `nil` if the sequence is `nil`) -/
 def tail : seq α → seq α
-| ⟨f, al⟩ := ⟨f.tail, λ n, al⟩
+| ⟨f, al⟩ := ⟨f.tail, λ _, al _⟩
```

Comment:
Does this work?
```suggestion
| ⟨f, al⟩ := ⟨f.tail, al⟩
```

Reply:
Doesn't work, one needs to lambda out a single term because of the tail